src/main/java/com/opencart/steps/LoginPageBL.java

```diff
@@ -69,4 +107,40 @@ private void clickOnLoginButton() {
         loginPage.getLoginButton().click();
     }
 
+    private void  clickOnForgottenPasswordButton() {
+        loginPage.getForgottenButton().click();
+    }
+
+    public void verifyLogin() {
+        String expectedText = "My Account";
+        Assert.assertEquals(accountDashboard.getMyAccount().getText(), expectedText, "You are not logged In!");
+    }
+
+    public void verifyLoginWithoutPassword() {
+        String expectedMessage = "Warning: No match for E-Mail Address and/or Password.";
+        Assert.assertEquals(loginPage.getWarning().getText(), expectedMessage, "You are logged in!");
+    }
+
+    public void verifyLoginWithoutEmail() {
+        String expectedMessage = "Warning: No match for E-Mail Address and/or Password.";
+        String expectedMessage1 = "Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.";
```

Comment:
Could you, please union expectedMessage and expectedMessage1  into one variable?